### PR TITLE
Fix failing LS tests after Removal of client validation

### PR DESCRIPTION
**Pull request by malinthar**
## Purpose
$subject

Fixes #<Issue Number>

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by malinthar** [comment]
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/38400

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/QNameRefCompletionUtil.java [hunk 1]
```diff
@@ -105,12 +107,20 @@ private static String getAlias(String alias) {
...
                                                 Predicate<Symbol> predicate) {
-        Optional<ModuleSymbol> module = ModuleUtil.searchModuleForAlias(context, 
+        Optional<ModuleSymbol> module = ModuleUtil.searchModuleForAlias(context,
                 QNameRefCompletionUtil.getAlias(qNameRef));
-        return module.map(moduleSymbol -> moduleSymbol.allSymbols().stream()
-                .filter(predicate)
-                .collect(Collectors.toList()))
+        List<Symbol> symbols = module.map(moduleSymbol -> moduleSymbol.allSymbols().stream()
+                        .filter(predicate)
+                        .collect(Collectors.toList()))
```
Comment (LakshanWeerasinghe) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/QNameRefCompletionUtil.java:114]:
```suggestion
                        .filter(predicate)
                        .collect(Collectors.toList()))
```
We can remove this additional spaces.

**Review by mohanvive — APPROVED** [review]

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java [hunk 2]
```diff
@@ -439,6 +438,8 @@ protected List<LSCompletionItem> getModuleCompletionItems(BallerinaCompletionCon
             completionItems.add(new StaticCompletionItem(ctx, item, StaticCompletionItem.Kind.MODULE));
         });
 
+        //Add prefixes of client declarations
+        completionItems.addAll(this.getClientDeclarationCompletionItems(ctx));
```
Comment (sanjana) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java:442]: Extra newline can be removed.
```suggestion
completionItems.addAll(this.getClientDeclarationCompletionItems(ctx));
```

**Review by sanjana — COMMENTED** [review]
Minor change only. Everything else LGTM.

**Pull request closed without merging**